pkg/master/master.go

```diff
@@ -567,12 +567,13 @@ func (m *Master) init(c *Config) {
 
 		"podTemplates": podTemplateStorage,
 
-		"replicationControllers": controllerStorage,
-		"services":               service.NewStorage(m.serviceRegistry, m.endpointRegistry, serviceClusterIPAllocator, serviceNodePortAllocator),
-		"endpoints":              endpointsStorage,
-		"nodes":                  nodeStorage,
-		"nodes/status":           nodeStatusStorage,
-		"events":                 eventStorage,
+		"replicationControllers":        controllerStorage,
+		"replicationControllers/status": controllerStatusStorage,
```

Comment:
Now that we have dropped the old name, we can normalize the internal
constants.  It would probably reduce confusion for code reviewers.

On Fri, Oct 9, 2015 at 12:16 PM, Brian Grant <notifications@github.com>
wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/14672#discussion_r41648501>
> :
>
> > @@ -567,12 +567,13 @@ func (m *Master) init(c *Config) {
> >
> >  		"podTemplates": podTemplateStorage,
> >
> > -		"replicationControllers": controllerStorage,
> > -		"services":               service.NewStorage(m.serviceRegistry, m.endpointRegistry, serviceClusterIPAllocator, serviceNodePortAllocator),
> > -		"endpoints":              endpointsStorage,
> > -		"nodes":                  nodeStorage,
> > -		"nodes/status":           nodeStatusStorage,
> > -		"events":                 eventStorage,
> > +		"replicationControllers":        controllerStorage,
> > +		"replicationControllers/status": controllerStatusStorage,
>
> Ugly, but true.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14672/files#r41648501>.
>